packages/contracts/deploy/009-ZKVerifier.ts

```diff
@@ -0,0 +1,19 @@
+import '@nomiclabs/hardhat-ethers'
+import { DeployFunction } from 'hardhat-deploy/dist/types'
+
+import { deploy } from '../src/deploy-utils'
+
+const deployFn: DeployFunction = async (hre) => {
+  await deploy(hre, 'ZKVerifier', {
+    args: [
+      hre.deployConfig.zkVerifierHashScalar,
+      hre.deployConfig.zkVerifierM56Px,
+      hre.deployConfig.zkVerifierM56Py,
+    ],
+    isProxyImpl: true,
```

Comment:
Could you add `postDeployAction` to check state variables set correctly?

Reply:
These variables are internal, so can't check them.